src/kirby/components/empty-state/empty-state.component.tns.html

```diff
@@ -0,0 +1,10 @@
+<FlexboxLayout flexDirection="column" justifyContent="space-around">
```

Comment:
Any need to wrap in flexbox layout if there's only 1 element (the stacklayout) inside?

Reply:
This was to get the StackLayout to center vertically.

Reply:
Ok but then move that to the example component, it shouldn't be part of the component itself as far as I can see?

Reply:
If we don't put the StackLayout inside the FlexboxLayout the content will be at the top of the screen. This means that everybody needs to center it vertically.

Reply:
Fixed